lib/read-from-url.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
+const tmp = require('tmp');
+const fetch = require('minipass-fetch');
+const fs = require('fs');
+const debug = require('debug')('supported:read-from-url');
+
+const RAW_CONTENT_HOST = 'raw.githubusercontent.com';
+
+async function setupProjectPath(_url, rawFileHost) {
+  let tempDir = tmp.dirSync();
```

Comment:
do we autoclean-up when the tmp dir is no longer needed?

Reply:
I removed this, now all operations are done in memory